### ISSUE TITLE
Error while loading rule 'compat/compat': rule.create is not a function

### ISSUE THREAD

**Issue by mobilemind** [amilajack/eslint-plugin-compat#96]
Just thought you might want to know about this. I'm getting the error while above using eslint-plugin-compat via eslint v4.8.0. Maybe it's something I'm doing or maybe it is a bug.

The error is:

`Error while loading rule 'compat/compat': rule.create is not a function`

It seems to be associated with the line in my `.eslintrc.yml` that worked fine with earlier versions of eslint-plugin-compat.

my .eslintrc.yml contains:
````yaml
plugins:
- compat

rules:
 # compat
  compat/compat: 2
````

Here is a run of eslint with debug on

````bash
eslint --debug src/isitaws.js
  eslint:cli Running on files +0ms
  eslint:glob-util Creating list of files to process. +0ms
  eslint:ignored-paths Looking for ignore file in projects/OpenInlets +0ms
  eslint:ignored-paths Loaded ignore file projects/OpenInlets/.eslintignore +0ms
  eslint:ignored-paths Adding projects/OpenInlets/.eslintignore +0ms
  eslint:cli-engine Processing projects/OpenInlets/src/isitaws.js +0ms
  eslint:cli-engine Linting projects/OpenInlets/src/isitaws.js +0ms
  eslint:config Constructing config file hierarchy for projects/OpenInlets/src +0ms
  eslint:config Using .eslintrc and package.json files +0ms
  eslint:config Loading projects/OpenInlets/.eslintrc.yml +1ms
  eslint:config-file Loading YAML config file: projects/OpenInlets/.eslintrc.yml +0ms
  eslint:config-file Loading /usr/local/lib/node_modules/eslint/conf/eslint-recommended.js +338ms
  eslint:config-file Loading JS config file: /usr/local/lib/node_modules/eslint/conf/eslint-recommended.js +0ms
  eslint:config Using projects/OpenInlets/.eslintrc.yml +358ms
  eslint:config-ops Using config from partial cache +0ms
  eslint:config-ops Apply environment settings to config +1ms
  eslint:config-ops Creating config for environment browser +0ms
  eslint:config-ops Creating config for environment es6 +1ms
  eslint:config-ops Creating config for environment node +0ms
  eslint:linter Linting code for projects/OpenInlets/src/isitaws.js (pass 1) +0ms
Error while loading rule 'compat/compat': rule.create is not a function
TypeError: Error while loading rule 'compat/compat': rule.create is not a function
    at Object.keys.forEach.ruleId (/usr/local/lib/node_modules/eslint/lib/linter.js:936:44)
    at Array.forEach (<anonymous>)
    at Linter._verifyWithoutProcessors (/usr/local/lib/node_modules/eslint/lib/linter.js:876:35)
    at preprocess.map.textBlock (/usr/local/lib/node_modules/eslint/lib/linter.js:1005:35)
    at Array.map (<anonymous>)
    at Linter.verify (/usr/local/lib/node_modules/eslint/lib/linter.js:1004:42)
    at Linter.verifyAndFix (/usr/local/lib/node_modules/eslint/lib/linter.js:1086:29)
    at processText (/usr/local/lib/node_modules/eslint/lib/cli-engine.js:175:32)
    at processFile (/usr/local/lib/node_modules/eslint/lib/cli-engine.js:219:18)
    at fileList.map.fileInfo (/usr/local/lib/node_modules/eslint/lib/cli-engine.js:537:20)
````

**Comment by jimblue** [comment]
got the same pb

**Comment by anx-ckreuzberger** [comment]
Same here

**Comment by exarus** [comment]
I've suggested a pull request that hopefully should fix the issue. Any feedback is welcome.

**Comment by amilajack** [comment]
Fixed this in `2.0.1`. @exarus thanks for the fix 🎉 🎉 🎉
